test/index.js

```diff
@@ -365,17 +367,76 @@ describe('Blockstore', () => {
 });
 
 // TODO:
-// Create scenarios where chain splits occur to form competing brances
-// Difficult with current chain provided by chainmanager as this is actual hardcoded
+// Create scenarios where chain splits occur to form competing branches
+// Difficult with current chain provided by chainmanager as this is actually hardcoded
 // Dash testnet headers which requires significant CPU power to create forked chains from
 
 describe('MerkleProofs', () => {
   it('should validate tx inclusion in merkleblock', () => {
-    const merkleBlock = new dashcore.MerkleBlock(merkleData.merkleBlock);
-    const validTx = '45afbfe270014d5593cb065562f1fed726f767fe334d8b3f4379025cfa5be8c5';
+    merkleBlock = new MerkleBlock(merkleData.merkleBlock);
+    const validTx = 'c5e85bfa5c0279433f8b4d33fe67f726d7fef1625506cb93554d0170e2bfaf45';
     const invalidTx = `${validTx.substring(0, validTx.length - 1)}0`;
-
     merkleProofs.validateTxProofs(merkleBlock, [validTx]).should.equal(true);
     merkleProofs.validateTxProofs(merkleBlock, [invalidTx]).should.equal(false);
   });
 });
+
+describe('Transaction validation', () => {
+  before(() => {
```

Comment:
It’s better to do that before each test due to test isolation principle (one of the tests might change it and affects others).

Reply:
agree.